src/Core/Resolvers/SqlPaginationUtil.cs

```diff
@@ -102,7 +102,9 @@ private static JsonObject CreatePaginationConnection(JsonElement root, Paginatio
             {
                 // parse *Connection.endCursor if there are no elements
                 // if no after is added, but it has been requested HotChocolate will report it as null
-                if (returnedElemNo > 0)
+                // Need to validate we have an extra element, because otherwise there is no next page
```

Comment:
The comment above this doesn't really make sense. Is it still applicable?

> // parse *Connection.endCursor if there are no elements
> // if no after is added, but it has been requested HotChocolate will report it as null

Reply:
I think what this comment is trying to say is that we you don't enter this conditional endCursor will be returned as null automatically, which is why we dont have to manually add it to the connection. I can change that language to try and be more clear.